backend/metering_billing/webhooks.py

```diff
@@ -128,6 +171,49 @@ def invoice_past_due_webhook(invoice, organization):
             )
 
 
+def subscription_created_webhook(subscription, subscription_data=None):
+    from api.serializers.model_serializers import SubscriptionRecordSerializer
+    from metering_billing.models import WebhookEndpoint
+
+    if SVIX_CONNECTOR is not None:
+        endpoints = (
+            WebhookEndpoint.objects.prefetch_related("triggers")
+            .filter(triggers__trigger_name=WEBHOOK_TRIGGER_EVENTS.SUBSCRIPTION_CREATED)
+            .distinct()
+        )
+        if endpoints.count() > 0:
+            svix = SVIX_CONNECTOR
+            now = str(now_utc())
+            payload = (
+                subscription_data if subscription_data else SubscriptionRecordSerializer(subscription).data
+            )
+            payload = make_all_decimals_floats(payload)
+            payload = make_all_dates_times_strings(payload)
+            response = {
+                "event_type": WEBHOOK_TRIGGER_EVENTS.SUBSCRIPTION_CREATED,
+                "payload": payload,
+            }
+            event_id = (
+                slugify(str(subscription.customer))
```

Comment:
I'm not sure which of the subscription fields would be appropriate for the `event_id`, so I just picked a couple.

Reply:
A good one could be subscription_record_id (a UUID field)... unfortunately we created subscriptions when we had a slightly different model structure in the backend so they're referred to as subscription records everywhere.